cas/remote.go

```diff
@@ -93,13 +153,40 @@ func (r Remote) Download(ds Store) (*Remote, error) {
 		return nil, fmt.Errorf("bad HTTP status code: %d", res.StatusCode)
 	}
 
-	key, err := ds.WriteACI(reader)
+	aciTempFile, err := ioutil.TempFile("", "")
```

Comment:
nit, `aciTempFile` is a slightly leaky name; just use `acif` or similar (as `downloadSignatureFile` does below)